lib/libraries/index.js

```diff
@@ -222,8 +222,15 @@ function defineC3(){
           this.dataset.matrix = [this.dataset.matrix[0], ...calculateRange(this.dataset.matrix)];
         }
 
-        if (!!this.config.clearOnRender && options.data.columns.length > 0) {
-          options.data.columns.splice(1);
+        if (!!this.config.clearOnRender && options.data.columns.length > 0 && this.dataset && this.dataset.meta) {
```

Comment:
please remove `> 0` from if statement
`!!this.config.clearOnRender && options.data.columns.length &&`